src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -763,6 +767,36 @@ private Vector(ref Register existingRegister)
         {
             this.register = existingRegister;
         }
+
+        /// <summary>
+        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
+        /// </summary>
+        [Intrinsic]
+        public unsafe Vector(Span<T> values)
```

Comment:
Is this method still `unsafe`?

Reply:
Removing it. It is left over from the earlier implementation in CoreFx code base that was on lines of constructor taking arrays, i.e. one which used pointers.